src/sidebar/components/group-list-section.js

```diff
@@ -1,23 +1,69 @@
 'use strict';
 
-// @ngInject
-function GroupListSectionController() {
-  this.isSelectable = function(groupId) {
-    const group = this.sectionGroups.find(g => g.id === groupId);
-    return !this.disableOosGroupSelection || group.isScopedToUri;
-  };
+const { Fragment, createElement } = require('preact');
+const propTypes = require('prop-types');
+
+const GroupListItem = require('./group-list-item');
+const GroupListItemOutOfScope = require('./group-list-item-out-of-scope');
+
+/**
+ * A labeled section of the groups list.
+ */
+function GroupListSection({
+  analytics,
+  disableOosGroupSelection,
+  heading,
+  sectionGroups,
+  store,
+}) {
+  const isSelectable = groupId =>
+    !disableOosGroupSelection ||
```

Comment:
This is a configuration parameter for notifying the controller that out of scope groups should not be selectable. This is necessary because Currently Viewing groups may be out of scope and still selectable whereas My Groups groups that are out of scope may not be selectable.

Reply:
It feels like this may be a bit of state data that belongs on the group objects themselves? Possibly?

Reply:
I actually wonder if this is strictly necessary anymore though. Since the isScopedToUri logic is no longer dependent on enforced we might be able to remove this config and just rely on enforcement and isScopedToUri now.

Reply:
const isSelectable = !group.scopes.enforced || group.isScopedToUri 

That being said you'll need to rebase to master to see this change in logic of the isScopedToUri.

Reply:
Ah, thanks. I'll take a look when I rebase.